src/main/java/com/intuit/graphql/orchestrator/utils/XtextToGraphQLJavaVisitor.java

```diff
@@ -522,6 +523,7 @@ public GraphQLSchemaElement caseDirectiveDefinition(DirectiveDefinition object)
 
   public void createBuiltInDirectives() {
     directiveDefinitions.put(DEPRECATED_DIRECTIVE.getName(), DEPRECATED_DIRECTIVE);
+    directiveDefinitions.put(KEY_DIRECTIVE.getName(), KEY_DIRECTIVE);
```

Comment:
why is this needed if you are loading the definition from the file?

Reply:
removed old code